scripts/e2e.ts

```diff
@@ -85,17 +84,18 @@ class E2ETestRunner extends events.EventEmitter {
     });
   }
 
-  #rendererProcess: null | childProcess.ChildProcess = null;
   /**
    * Start the renderer process.
    */
-  startRendererProcess(): Promise<void> {
-    this.#rendererProcess = this.spawn('Renderer process',
-      'node', 'node_modules/nuxt/bin/nuxt.js',
-      '--hostname', 'localhost',
-      '--port', this.rendererPort.toString(), buildUtils.rendererSrcDir);
-
-    return Promise.resolve();
+  buildRenderer(): Promise<void> {
+    process.env.VUE_CLI_SERVICE_CONFIG_PATH = 'pkg/rancher-desktop/vue.config.js';
+
+    return buildUtils.spawn(
+      'node_modules/.bin/vue-cli-service',
+      'build',
+      '--skip-plugins',
+      'eslint',
```

Comment:
Since this is shared with `build.ts`, should we move it to `build-utils.ts`? Or should we _not_ `--skip-plugins eslint` in the production build?

Reply:
We manually invoke lint most everywhere we run any build scripts. Skipping the vue-cli plugin reduces build times and redundancy here.